machine/dni/dni_3048up/INSTALL

```diff
@@ -46,12 +46,12 @@ Step 2 -- Install ONIE kernel (dni_3048up-r1.u-boot dni_3048up-r1.itb)
 
 Copying the image down using TFTP and flash to the SPI flash::
 
-  =>tftp $loadaddr dni_3048up-r1.u-boot
+  =>tftp $loadaddr onie-dni_3048up-r1.bin.uboot+env
```

Comment:
Install instructions still seem to be out of sync.  
Can we make sure INSTALL document reflects with real example for generating binaries, expected binaries after the build and install instructions

Ex:
   $ cd build-config
   $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all
 
 When complete, the ONIE binaries are located in
 ``build/images``::
 
   -rw-r--r-- 3604480 May  7 11:38 onie-dni_3048up-r1.bin  ---> Two binaries should have been generated, one for u-boot and one for ONIE.

2)
 Install ONIE kernel (dni_3048up-r1.u-boot dni_3048up-r1.itb) --> Update with correct names